dist/stat_data.py

```diff
@@ -105,6 +100,11 @@ class DhandleStat(Stat):
     prefix = 'data-handle'
     def __init__(self, name, desc, flags=''):
         Stat.__init__(self, name, DhandleStat.prefix, desc, flags)
+class EvictCacheWalkStat(Stat):
```

Comment:
@raviprakashgiri29 we've changed the name of this stat group but there's no change to the list of stats or their text descriptions. 
Does this impact your current T2 work and the `WiredTiger Eviction Cache State` category?

Reply:
@ajmorton Thanks for the heads up. This should not have any impact on t2 since the 'cache_walk' prefix and stat text descriptions remain unchanged.